.release-automation/utilities.py

```diff
@@ -20,7 +21,7 @@ def upload_jars(configs):
     print("Upload jars to signing server...")
     jar_list = []
     for module_name in configs["moduleNames"]:
-        module_folder = os.path.join(configs["srcFolder"], module_name, configs["releaseVersion"])
+        module_folder = get_module_folder(configs, module_name)
         module_jars = glob.glob(os.path.join(module_folder, "*.jar"))
```

Comment:
Consider moving line#25, line#70 and line#98 into method `get_module_folder`? Probably rename method to `get_target_files`.